packages/ui/src/components/CanvasContextMenu.tsx

```diff
@@ -36,35 +36,36 @@ export function CanvasContextMenu(props) {
   const store = useContext(RepoContext);
   if (!store) throw new Error("Missing BearContext.Provider in the tree");
 
-  const isGuest = useStore(store, (state) => state.role === "GUEST");
+  const editMode = useStore(store, (state) => state.editMode);
+  const editing = editMode === "edit";
   return (
     <Box sx={paneMenuStyle(props.x, props.y)}>
       <MenuList className="paneContextMenu">
-        {!isGuest && (
+        {editing && (
```

Comment:
Is it better to control the parent <Box> by `editing`, rather than repeating the `editing` check on each <MenuItem>?

Reply:
There could be some context menu items in "view" mode as well.